Lib/pyclbr.py

```diff
@@ -45,39 +44,57 @@
 import tokenize
 from token import NAME, DEDENT, OP
 
-__all__ = ["readmodule", "readmodule_ex", "Class", "Function"]
+__all__ = ["readmodule", "readmodule_ex", "Object", "Class", "Function"]
 
-_modules = {}                           # cache of modules we've seen
+_modules = {}  # Initialize cache of modules we've seen.
 
-# each Python class is represented by an instance of this class
-class Class:
-    '''Class to represent a Python class.'''
-    def __init__(self, module, name, super, file, lineno):
+
+class Object:
```

Comment:
The docstrings and comments are already part of your branch, and visible when I hit [files] at the top.  I have already done the _Object changes, the _create_tree docstring and setup for the revised test.  I am working on the hard part -- a revised comparison function.